test/integration/associations/belongs-to-many.test.js

```diff
@@ -493,23 +486,18 @@ describe(Support.getTestDialectTeaser('BelongsToMany'), function() {
         }
       });
 
-      return this.UserTask.sync({ force: true})
-      .bind(this)
-      .then(function() {
-        return Promise.join(
-          this.Task.create().then(function (task) {
-            return user.addStartedTask(task, {
-              started: true
-            });
-          }),
-          this.Task.create().then(function (task) {
-            return user.addStartedTask(task, {
-              started: true
-            });
-          })
-        );
-      })
-      .then(function () {
+      return Promise.join(
```

Comment:
@sushantdhiman Can you explain what this change did? Was it a fix of the test, unrelated to the `foreignKey`/`as` change?